### DISCUSSION TITLE
How to Delete an Organization [Solved]

### DISCUSSION THREAD

**Discussion by flyingfishflash** [zitadel/zitadel#4949] [Q&A] [answered]
Sorry if this is an obvious question, but for the life of me I cannot find how to delete an organization. EG I created an organization for testing. How do I now delete this organization (beyond deactivating it)?

Thanks

**Comment by eliobischof** [comment, marked as answer]
> Sorry if this is an obvious question, but for the life of me I cannot find how to delete an organization. EG I created an organization for testing. How do I now delete this organization (beyond deactivating it)?
> 
> Thanks

Since ZITADEL v2.14.0 you can delete organizations [via API](https://zitadel.com/docs/apis/proto/management#removeorg).
Since v2.17.0 you can also delete them via GUI.
![image](https://user-images.githubusercontent.com/12727842/209712431-22d673d2-38ed-4930-bc28-2383c9c83100.png)

The version on zitadel.cloud is v2.16.2, ATM.

**Reply by flyingfishflash** [reply]
Thank you! I am self-hosting 2.16.2 via container, so will pull in the UI update eventually.

**Reply by fforootd** [reply]
FYI I moved this to Q&A and set it to answered.
